src/pages/Login.vue

```diff
@@ -22,10 +22,18 @@
             bg-color="white"
             filled
             class="q-px-sm form-input col-xs-12 col-md-12"
-            type="password"
+            :type="isPwd ? 'password' : 'text'"
```

Comment:
Please rename `isPwd` to `showPassword`. Not that you'll have to change the order here and set the default in data to false

Reply:
What do you mean by `data` to false?
If I set `showPassword` to false, you can see the plain text.

Reply:
with `change the order` I meant
`:type="showPassword ? 'text' : 'password'"`

Reply:
Hmm.. dunno if that change is so much necessary.

Reply:
See the changes